src/vm/core.zig

```diff
@@ -322,11 +322,10 @@ pub const CairoVM = struct {
         instruction: *const instructions.Instruction,
     ) !OperandsResult {
         var op_res = OperandsResult.default();
-
         op_res.res = null;
 
         op_res.dst_addr = try self.run_context.computeDstAddr(instruction);
-        const dst_op = try self.segments.memory.get(op_res.dst_addr);
+        const dst_op = try self.segments.memory.get2(op_res.dst_addr);
```

Comment:
this change addresses the error that happens as reported in #205, i implemented this change as a silly extra method `get2` because there are a lot of tests that fail if this change is made directly on the `get` method and I'd like to sanity check before making a deep change and refactoring a lot of tests

https://gist.github.com/jobez/f2f70345b76c4da4a573b813df0e0ee4